12-connectivity.tf

```diff
@@ -103,12 +47,12 @@ resource "azurerm_route" "additional_route" {
   next_hop_in_ip_address = each.value.next_hop_type != "VirtualAppliance" ? null : each.value.next_hop_in_ip_address
 }
 
-resource "azurerm_subnet_route_table_association" "aks_00" {
-  route_table_id = azurerm_route_table.route_table.id
-  subnet_id      = azurerm_subnet.aks_00_subnet.id
-}
+resource "azurerm_subnet_route_table_association" "route_table" {
+  for_each = { for subnet in var.subnets : subnet.name => subnet
+    if subnet.route_table == true
```

Comment:
the default should be setup to include what we expect, i.e. a route table on each subnet pointing to the hub, with an opt-out if required

🚫 Blocking

Reply:
Sure - will swap to `!= false`